dxcint/dxcint/Terraform.py

```diff
@@ -140,12 +140,13 @@ def _build_asset(self, language: str) -> str:
         return asset_obj.get("id")
 
     def _create_asset_spec(self, language: str) -> Dict:
-        spec_exports = [x.export_spec() for x in self._dependencies].remove(None)
+        # spec_exports = [x.export_spec() for x in self._dependencies].remove(None) # this line doesn't do anything
```

Comment:
The [].remove method does not have a return value, I reintroduced the correct behaviour in APPS-1432 when it was needed, I can put it here as well, it will create merge conflicts though.